main.cc

```diff
@@ -662,12 +663,14 @@ int main(int ac, char** av) {
             static sharded<auth::service> auth_service;
             static sharded<db::system_distributed_keyspace> sys_dist_ks;
             static sharded<db::view::view_update_generator> view_update_generator;
+            static sharded<cql3::cql_config> cql_config;
+            cql_config.start().get();
```

Comment:
Should we stop cql_config at some point?

Reply:
Since we don't shut down everything, this carries the risk that system queries executed after cql_config is shut down will reference deallocated memory.

Reply:
(clean shutdown is an all-or-nothing thing)